### ISSUE TITLE
[Quest] Ahune, the Frost Lord

### ISSUE THREAD

**Issue by Golaman2** [dalaranwow/dalaran-wow#5751]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Quest starts from Earthen Ring Elder in Shattrath City. Should be delivered in Slave Pens but Numa Cloudsister (NPC) is nowhere to be found.

**Expected behaviour**: Tell us what should happen instead
NPC must be inside the Slave Pens in Coilfand Reservoir
**Steps to reproduce the problem**:

1. Searched the NPC after I queued for Ahune event boss
2. Searched the NPC after running in the instance
3. Both ways, there is no NPC

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
NPC ID is 25710

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

You need to use the stone, in order to summon Ahune.

**Comment by Golaman2** [comment]
done that... Quest ID is 11955 and it is a pre-quest to the quest that need you to kill/summon Ahune..
